css/swc.css

```diff
@@ -18,11 +18,6 @@ div.chapter h2 {
     font-style: italic;
 }
 
-/* Objectives and key points */
-.objectives, .keypoints {
```

Comment:
The deleted instance of `.objectives` was redundant; it is repeated at line 116.